platform/ui/src/elements/Icon/getIcon.js

```diff
@@ -85,8 +85,12 @@ import trash from './icons/trash.svg';
 import unlink from './icons/unlink.svg';
 import user from './icons/user.svg';
 import youtube from './icons/youtube.svg';
+import eye from './icons/eye.svg';
+import eyeClosed from './icons/eye-closed.svg';
 
 const ICONS = {
+  eye,
+  'eye-closed': eyeClosed,
```

Comment:
@dannyrb Just a note, but not related to this PR, as this one is generic. I don't think we have a good way of adding icons via extensions?

They need to be registered to the Icon list and then be usable everywhere?

For example when adding tools to the tool bar they just reference an icon's name from this list, so you can't even inject an element at that level currently?